testutil/crypto/crypto.go

```diff
@@ -31,10 +33,16 @@ func NewCryptoIdentityFromBytesSeed(seed []byte) CryptoIdentity {
 	return CryptoIdentity{PV: privKey}
 }
 
-func NewCryptoIdentityFromIntSeed(i int) CryptoIdentity {
-	iUint64 := uint64(i)
+func NewCryptoIdentityFromRandSeed() CryptoIdentity {
+	b := make([]byte, 8)
+	_, _ = rand.Read(b)
+	seed := binary.BigEndian.Uint64(b)
+	return NewCryptoIdentityFromIntSeed(seed)
+}
+
+func NewCryptoIdentityFromIntSeed(i uint64) CryptoIdentity {
 	seed := []byte("AAAAAAAAabcdefghijklmnopqrstuvwx") // 8+24 bytes
-	binary.LittleEndian.PutUint64(seed[:8], iUint64)
+	binary.LittleEndian.PutUint64(seed[:8], i)
 	return NewCryptoIdentityFromBytesSeed(seed)
 }
```

Comment:
I would prefer a separate function `FromUint64Seed` if it's necessary. I used `int` as the arg to begin with because it's the easiest type to use when writing tests (in loops and such).